### PR TITLE
Check in R docs

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: api/r/property_label.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+# Show Property Labels of Node(s)
+
+## `get_property_labels(dcids, out=True)`
+
+Returns the labels of properties defined for the given node DCIDs
+
+**Arguments**
+
+*    `dcids` (`vector` of strings) - DCIDs identifying nodes to query.
```
Comment (beets) [inline api/r/property_label.md]: spacing nit

File: api/r/property_label.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+> set_api_key(YOUR_API_KEY_HERE)
+> get_property_labels(c('geoId/06', 'geoId/08'))
+$`geoId/06`
+[1] "containedInPlace" "geoId"            "kmlCoordinates"  
+[4] "name"             "provenance"       "typeOf"          
+
+$`geoId/08`
+[1] "containedInPlace" "geoId"            "kmlCoordinates"  
+[4] "name"             "provenance"       "typeOf"
+
```
Comment (beets) [inline api/r/property_label.md]: nit: remove empty lines here (and below)

File: api/r/property_label.md (hunk 1 above)
Comment (tjann) [reply api/r/property_label.md]: fixed

File: api/r/property_label.md (hunk 2 above)
Comment (tjann) [reply api/r/property_label.md]: R returns new lines for output but I agree, not needed :)

**Pull request merged**